ITISFoundation/osparc-simcore
# 🎨 web-api `services` api response  includes manifest info (part 5)

## What do these changes do?
Connects manifest to `services` web-api response (requirement for UI)

The service metadata returned by the API is composed of two intersecting parts, namely a "published" and an "editable" parts. The first comes from the manifest (i.e. metadata from services in the docker registry and function-services) and the second comes from the `services_meta_data` table. 

This PR completes the service meta-data returned in the API by accessing and merging the "published" part. This part includes fields like inputs/outputs, etc

#### Extras

 - 🔨 prints actual docker-compose specs used in the tests
 
## Related issue/s

- Part of #5964 

## How to test

- Main driving test `services/catalog/tests/unit/with_dbs/test_services_services_api.py`
- Manual testing `/dev/catalog/services/*` entrypoints using swagger site 
    - set `WEBSERVER_DEV_FEATURES_ENABLED=1` in `.env`
    - deploy
    - Execute entrypoint and should receive a complete answer
![image](https://github.com/user-attachments/assets/b049f9a3-86c2-40e1-98de-ee7f0362c1e3)
   

Following PR will address:
- evaluate and add `compatibility` field in response body


## Dev-ops checklist
